src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -176,14 +176,15 @@ public void InvalidActionSpecifiedForARole(string dbPolicy, Config.Operation act
         [DataRow("1 eq @item.col2", Config.Operation.Read, false, DisplayName = "Database Policy defined for Read passes")]
         [DataRow("2 eq @item.col3", Config.Operation.Update, false, DisplayName = "Database Policy defined for Update passes")]
         [DataRow("2 eq @item.col3", Config.Operation.Delete, false, DisplayName = "Database Policy defined for Delete passes")]
-        public void AddDatabasePolicyToCreateOperationPermission(string dbPolicy, Config.Operation action, bool errorExpected)
+        public void AddDatabasePolicyToCreateOperationForMySql(string dbPolicy, Config.Operation action, bool errorExpected)
```

Comment:
Does it work for PostgreSql?

Reply:
No, will summarise it to better indicate that.